SimPEG/potential_fields/magnetics/simulation.py

```diff
@@ -196,8 +196,8 @@ def Jtvec(self, m, v, f=None):
     @property
     def fieldDeriv(self):
 
-        if self.chi is None:
-            self.model = np.zeros(self.G.shape[1])
+        if getattr(self, 'chi', None) is None:
+            self.model = m
```

Comment:
I would guess the `self.model = m` line would probably not work right here, as we never pass `m` to fieldDeriv (which might not be a good thing).

Reply:
I noticed that, it is already been fixed.

Reply:
For the same reason, I kept the `if getattr(self, 'chi'` ` for that specific case.